src/theme/shadows.ts

```diff
@@ -1,8 +1,8 @@
 export const shadows = {
   xs: '0px 0px 4px 1px rgba(0, 0, 0, 0.16)',
-  'white-xs': '0px 0px 4px 1px rgba(125, 125, 125, 0.16)',
+  'white-xs': '',
   sm: '0px 2px 4px rgba(0, 0, 0, 0.16)',
-  'white-sm': '0px 2px 4px rgba(125, 125, 125, 0.16)',
+  'white-sm': '',
   md: '0px 4px 8px rgba(0, 0, 0, 0.16)',
-  'white-md': '0px 4px 8px rgba(125, 125, 125, 0.16)'
+  'white-md': ''
 }
```

Comment:
If these `white-*` ones are being made empty, then shouldn't they just be removed entirely?

Reply:
Thanks, removed in: [5830c56](https://github.com/WootingKb/wooting-macros/pull/194/commits/5830c56f2667220c804acb4b1dd463fcc54cf649)